go/api/glide_client.go

```diff
@@ -352,3 +352,18 @@ func (client *baseClient) LolwutWithOptions(opts options.LolwutOptions) (string,
 	}
 	return handleStringResponse(result)
 }
+
+// Returns a random existing key name from the currently selected database.
+//
+// Return value:
+//
+//	A random existing key name from the currently selected database.
+//
+// [valkey.io]: https://valkey.io/commands/randomkey/
+func (client *GlideClient) RandomKey() (Result[string],, error) {
```

Comment:
```suggestion
func (client *GlideClient) RandomKey() (Result[string], error) {
```
extra comma failing the compilation

Reply:
oops, typo
already removed